backend/test_nightly/conftest.py

```diff
@@ -150,3 +150,33 @@ def crawl_config_info(admin_auth_headers, default_org_id):
         if data["state"] == "complete":
             return (crawl_config_id, crawl_id, second_crawl_id)
         time.sleep(5)
+
+
+@pytest.fixture(scope="session")
+def error_crawl_id(admin_auth_headers, default_org_id):
+    crawl_data = {
+        "runNow": True,
+        "name": "Youtube crawl with errors",
+        "config": {
+            "seeds": [{"url": "https://www.youtube.com/watch?v=Sh-x3QmbRZc"}],
```

Comment:
Hm, wonder if we should do this as part of nightly test, or something more 'predictable', such as list of URLs with some being invalid?